.github/workflows/gradle.yml

```diff
@@ -3,21 +3,24 @@ name: Java & Go CI
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths-ignore:
+      - 'docs/**'
+      - 'github-actions/**'
+      - 'sechub-doc/**'
+      - 'sechub-website/**'
+      - 'sechub-webui/**'
+      - 'sechub-webui-solution/**'
     # We ignore everything where tag starts with v* - this is done by release build!
     tags-ignore:
       - v*
   pull_request:
-    # This workflow will run for all pull requests that target following branches
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths-ignore:
+      - 'docs/**'
+      - 'github-actions/**'
+      - 'sechub-doc/**'
+      - 'sechub-website/**'
+      - 'sechub-webui/**'
```

Comment:
Should the `sechub-webui` really be excluded? It's a deployable Spring Boot app like the others

Reply:
Good point :+1:  - suggestion added.

Reply:
sechub-webui removed from the exclude list